utils/storage_utils.py

```diff
@@ -53,9 +55,11 @@ def write_output_csv(start_time, end_time, cve_model_type, ecosystem, df, s3_upl
         "closed_at",
         "creator_name",
         "creator_url",
+        "ecosystem",
+        "title",
```

Comment:
Are these columns "title" and "body" available in this data frame already?

Reply:
Yes they are available